config/jobs/kubernetes-csi/csi-driver-host-path/manual-job-config.yaml

```diff
@@ -45,13 +45,13 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-      resources:
```

Comment:
I feel like a lot of these were generated by a script. Is that in here? It should be changed

Reply:
this one is a "manual" file. I have not edited the ones generated by scripts in this PR, that's where I started to run out of steam.